src/components/current-query/index.js

```diff
@@ -5,21 +5,25 @@ import moment from 'moment';
 import { LiveMessenger } from 'react-aria-live';
 import helpers from '../../helpers';
 
-
 // Create dumb component which can be configured by props.
-const FacetType = props => (
-  <button className="fs-applied-filters__filter" key={props.id} onClick={props.onClick}>
+const FacetType = ({ id, onClick, children }) => (
+  <button className="fs-applied-filters__filter" key={id} onClick={onClick} type="submit">
     <span className="fs-element-invisible">
       Remove filter
     </span>
-    {props.children}
+    {children}
   </button>
 );
+FacetType.propTypes = {
+  id: PropTypes.string.isRequired,
+  onClick: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
```

Comment:
I could probably remove `isRequired` and provide a default of `[]` instead. I think I did that in at least one other place.